whisper/whisperv6/peer_test.go

```diff
@@ -512,3 +513,47 @@ func waitForServersToStart(t *testing.T) {
 	}
 	t.Fatalf("Failed to start all the servers, running: %d", started)
 }
+
+func TestPureLightPeerHandshake(t *testing.T) {
```

Comment:
Those unit tests are nice, however I would like to see some more advanced tests that create servers and see if a light server gets disconnected.